app/src/main/java/org/mozilla/focus/home/HomeTilesManager.kt

```diff
@@ -66,7 +66,8 @@ class BundledTilesManager @VisibleForTesting constructor(
         }
 
         private fun loadBlacklist(context: Context): MutableSet<String> {
-            return context.getSharedPreferences(PREF_HOME_TILES, MODE_PRIVATE).getStringSet(BUNDLED_SITES_ID_BLACKLIST, mutableSetOf())
+            return context.getSharedPreferences(PREF_HOME_TILES, MODE_PRIVATE)
+                .getStringSet(BUNDLED_SITES_ID_BLACKLIST, null) ?: mutableSetOf()
```

Comment:
This might be a good place for a Sentry assertion.

Reply:
@Baron-Severin For what purpose? After this code change, there aren't any error states to record, afaict. The API is a little awkward, but the pattern works (tangential: maybe we should write an extension function to make this harder to mess up).